packages/components/src/FilterMultiSelect/subcomponents/SearchInput/SearchInput.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useMemo } from "react"
 import { v4 } from "uuid"
```

Comment:
Should we replace this with React's useId?

Reply:
Yes, but not in this PR 😄 
https://cultureamp.atlassian.net/browse/KDS-1673

Reply:
nice